FOSUserBundle.php

```diff
@@ -11,13 +11,14 @@
 
 namespace FOS\UserBundle;
 
-use Symfony\Component\HttpKernel\Bundle\Bundle;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
-use FOS\UserBundle\DependencyInjection\Compiler\ValidationPass;
-use FOS\UserBundle\DependencyInjection\Compiler\RegisterMappingsPass;
+use Doctrine\Bundle\CouchDBBundle\DependencyInjection\Compiler\DoctrineCouchDBMappingsPass;
 use Doctrine\Bundle\DoctrineBundle\DependencyInjection\Compiler\DoctrineOrmMappingsPass;
 use Doctrine\Bundle\MongoDBBundle\DependencyInjection\Compiler\DoctrineMongoDBMappingsPass;
-use Doctrine\Bundle\CouchDBBundle\DependencyInjection\Compiler\DoctrineCouchDBMappingsPass;
+use FOS\UserBundle\DependencyInjection\Compiler\LoginManagerPass;
+use FOS\UserBundle\DependencyInjection\Compiler\RegisterMappingsPass;
+use FOS\UserBundle\DependencyInjection\Compiler\ValidationPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\HttpKernel\Bundle\Bundle;
```

Comment:
You can completely revert the changes here (in fact you must at least remove the `LoginManagerPass`).